packages/rocketchat-lib/server/models/Subscriptions.coffee

```diff
@@ -312,7 +312,6 @@ class ModelSubscriptions extends RocketChat.models._Base
 		update =
 			$set:
 				alert: true
-				open: true
```

Comment:
Oh, I had no idea.  I couldn't find any docs about it and the code and comments didn't make it obvious.  So the current behavior just hides it until there is activity.  This was the intent?  So I assume people probably hid rooms that they were a member of but are not very active?